core/kotlinx-coroutines-core/test/internal/LockFreeTaskQueueLinearizabilityTest.kt

```diff
@@ -14,12 +14,13 @@ import kotlin.test.*
 
 @OpGroupConfigs(OpGroupConfig(name = "consumer", nonParallel = true))
 @Param(name = "value", gen = IntGen::class, conf = "1:3")
-class LockFreeMPSCQueueLinearizabilityTest : TestBase() {
-    private lateinit var q: LockFreeMPSCQueue<Int>
+class LockFreeTaskQueueLinearizabilityTest : TestBase() {
+    private var singleConsumer = false
```

Comment:
fun fact: mutating instance field from the test does not work because lintest creates new instance of the test every iteration.
So MPMC queue is tested in both tests

Reply:
Uh-oh!